.devcontainer/devcontainer.json

```diff
@@ -29,8 +29,7 @@
 		"ryuta46.multi-command",
 		"coolchyni.beyond-debug"
 	],
-	"initializeCommand": "docker pull ghcr.io/magma/magma/devcontainer:latest",
-	"image": "ghcr.io/magma/magma/devcontainer:latest",
+	"image": "ghcr.io/magma/magma/devcontainer:sha-3ed7bc8",
```

Comment:
I found that even with this explicit hash, vscode will still keep using existing devcontainers. In order to use a new container, you need to run "Remote Containers: Rebuild Containers" in vscode. Maybe we should add a line to the readme indicating that.

Reply:
I am not so much in favour of that, because this is not a Magma thing, but rather a "How to use VS code properly", and I am not a fan of duplicating or writing a VS code usage guide. This is best done by the VS code people in their documentation.

Reply:
I agree that we shouldn't accumulate too much documentation on third party tools. On the other hand, I think some basic pointers and hints are ok and helpful. Given that several people were suprised to learn that their devcontainers were never updated, it might still make sense to give a small hint in the docs.

Reply:
https://github.com/magma/magma/wiki/Contributing-Code-with-VSCode#known-issues-with-vscode :heavy_check_mark: